.github/actions/install-happy/action.yaml

```
@@ -22,10 +22,13 @@ runs:
       run: |
         set -ue
         cd $(mktemp -d)
+        if [ -z $HAPPY_VERSION ] || [ $HAPPY_VERSION == "latest" ]
+        then
+          HAPPY_VERSION=$(wget --quiet -O - https://api.github.com/repos/chanzuckerberg/happy/releases/latest | jq -r ".. .tag_name? // empty" | sed -e "s/^v//")
```

in case the latest tag has a prefix (ie, `api-v0.39.0`)
```suggestion
          HAPPY_VERSION=$(wget --quiet -O - https://api.github.com/repos/chanzuckerberg/happy/releases/latest | jq -r ".. .tag_name? // empty" | sed -e "s/^[a-z\-]*v//")
```